### PR TITLE
Standard T Distribution

### PR DESCRIPTION
**Pull request by YoshikawaMasashi**
split from #1177

### CODE REVIEW

**Comment by YoshikawaMasashi** [comment]
jenkins, test this please.

**Comment by YoshikawaMasashi** [comment]
jenkins, test this please.

**Comment by toslunar** [comment]
Jenkins, test this please.

File: cupy/random/distributions.py [hunk 1]
```diff
@@ -351,6 +351,35 @@ def standard_normal(size=None, dtype=float):
...
+        size (int or tuple of ints): The shape of the array. If ``None``, a
+            zero-dimensional array is generated.
+        dtype: Data type specifier. Only :class:`numpy.float32` and
+            :class:`numpy.float64` types are allowed.
+
+    Returns:
+        cupy.ndarray: Samples drawn from the standard Student's t distribution.
+
+    .. seealso::
+        :func:`cupy.random.RandomState.standard_t`
```
Comment (toslunar) [inline cupy/random/distributions.py]: Remove this line.

**Comment by toslunar** [comment]
Jenkins, test this please.

**Pull request merged**